src/charwise.rs

```diff
@@ -779,6 +779,10 @@ impl CharwiseDoubleArrayAhoCorasick {
     ///
     /// * `source` - A source slice.
     ///
+    /// # Returns
+    ///
+    /// A tuple of the automaton and the tail of the slice.
```

Comment:
```suggestion
    /// A tuple of the automaton and the slice not used for the deserialization.
```
I suggest a more specific description.